tools/c7n_azure/tests/test_deployment_units.py

```diff
@@ -57,7 +59,7 @@ def test_app_insights(self):
         self._validate(unit, params)
 
     def test_storage_account(self):
-        params = {'name': 'custodianaccount47182748',
+        params = {'name': 'custodianaccount47182745',

```

Comment:
outside of this PR scope: should we replace hardcoded values with uniqueStr() approach?

Reply:
Yeah I found those pretty weird, and in this case broken.  I was trying to avoid rewriting all tests beyond what was necessary, but this isn't ideal.  I'll file a bug.